### PR TITLE
TrustSwap - Milestone 1 & 2 Combined

### PR DESCRIPTION
**Pull request by stebansaa**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/eosnetworkfoundation/grant-milestones/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form](https://forms.gle/wLuAzXKa9qYrZQob9) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the EOS account name provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/eosnetworkfoundation/grant-framework/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/eosnetworkfoundation/grant-framework/pull/42

### CODE REVIEW

**Comment by stebansaa** [comment]
Thank you!

File: deliveries/trustswap.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Deliverables
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+|0a.| License |	https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt | MIT |
+|0b.| Documentation | https://docs.frogge.finance | Documentation |	We will provide both inline documentation of the code and a basic tutorial that explains how to configure TrustEVM and use TrustSwap
+|0c.| Testing Guide	| https://github.com/evm20/v2-core/tree/master/test | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.
+|0d.| Front-end Deployment | https://github.com/evm20/interface | We will provide a repository with instructions on how to deploy the frontend.
+|0e.| Article | https://docs.frogge.finance/article | We will publish an article that explains how the different components of the platform are connected to make the platform work
```
Comment (DenisCarriere) [inline deliveries/trustswap.md:22]: Link should be: https://docs.frogge.finance/articles/article

**Review by DenisCarriere — APPROVED** [review]
Reviewed & looks good to me 👍

File: deliveries/trustswap.md (hunk 1 above)
Comment (stebansaa) [reply deliveries/trustswap.md:22]: Thank you, will wait for Bart review, then fix the broken link.

**Comment by stebansaa** [comment]
is there something else needed from my part to complete this?

File: deliveries/trustswap.md (hunk 1 above)
Comment (wanderingbort) [reply deliveries/trustswap.md:22]: agreed, though it was easily discoverable through docs.

**Review by wanderingbort — APPROVED** [review]

**Comment by ABurton22** [comment]
Submitted for payment processing

Amy Burton

**Pull request merged**